### ISSUE TITLE
Additional Accessibility Features plugin translate (reopen)

### ISSUE THREAD

**Issue by LinelabCZ** [joomla/joomla-cms#45415]
Please check media/vendor/accessibility/js/ accessibility.js, min, gzip
`this.options.icon.useEmojis?"♲":"**refresh**`
close
accessible

These words are part of JavaScript and cannot be translated. Additionally, Wave reports a Contrast Error: Very low contrast for Access menu box. See attached.

![Image](https://github.com/user-attachments/assets/120bc377-7596-4c76-9f3e-143ab0ff2331)

**Comment by QuyTon** [comment]
I am still unsure what the issue is. Are you referring to the tooltip?

![Image](https://github.com/user-attachments/assets/557de7d7-939c-470c-8597-765386019fa9)

**Comment by LinelabCZ** [comment]
Yes, I mean the labels. There are several labels like "refresh", "close", "accessible". These are basic menu control elements. If the user uses text-to-speech and doesn't speak English, the reader will read the English terms. The user doesn't understand words like "accessible".

**Comment by brianteeman** [comment]
I assume from your screen shot that you are using the Czech language?

I cannot replicate your problem in Czech

![Image](https://github.com/user-attachments/assets/53136e5b-c343-4f01-8d42-905744157692)

**Comment by brianteeman** [comment]
I see what you are saying now.

The good news is that the title is translated and that is what accessibility tools will use

![Image](https://github.com/user-attachments/assets/bcdc7723-7576-42fb-8ed1-4b1ea35fe36c)

The bad news is that of course it could be improved but the plugin is using a very outdated third party library that will require considerable work but the accessibility team are still deciding what to do with the plugin so I wont commit to spending time on it until they make a decision

**Comment by LinelabCZ** [comment]
Thank you for the message, but I will repeat once again: If the user is using text-to-speech, the English name is preferred, and furthermore, the pronunciation is incorrect. Instead of /kləʊs/, it pronounces the letters c l o s e.  :-)

**Comment by brianteeman** [comment]
That was not my experience using Narrator - i didnt test other screen readers

But none of that changes the facts that this will NOT be fixed until the accessibility team make a decision.

**Comment by LinelabCZ** [comment]
I understand. Just to clarify, a TTS system that is trained only for a native language doesn't recognize when an English word appears in the text.
It reads it according to the pronunciation rules of that language, in the case of Czech, for example, it might read "Close" as "close".  I’ll modify the file media/vendor/accessibility/js/accessibility.js, that’s not a problem. 

Still, thank you for your answer.
